cfme/tests/infrastructure/test_vm_retirement_rest.py

```diff
@@ -3,7 +3,7 @@
 import fauxfactory
 
 from cfme.rest import a_provider as _a_provider
-from utils.version import appliance_is_downstream
+from utils.version import current_version
```

Comment:
@mfalesni I think yes and it should be replace to store.current_appliance.version